src/graphql/typeDefs/Event.ts

```diff
@@ -172,6 +172,13 @@ export default gql`
     submissionId: String!
   }
 
+  type TransactionMessageEvent {
+    type: EventType!
+    transactionHash: String!
+    message: String!
+    colonyAddress: String!
+  }
```

Comment:
i think we also need to return person which added comment and time when this person added comment